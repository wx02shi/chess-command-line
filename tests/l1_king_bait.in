game computer1 computer1 
setup
+ K c4
= black
+ P b4
+ P b3
+ P c3
+ P d3
+ P d4

+ Q a4
+ Q a3
+ Q b2
+ Q c2
+ Q d2
+ Q e3
+ Q e4

+ K h2
done
move
