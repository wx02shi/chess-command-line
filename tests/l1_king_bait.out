8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _K_ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  pK_ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  pK_ _ _
3 _p_ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  pK_ _ _
3 _pp _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  pK_ _ _
3 _ppp_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  pKp _ _
3 _ppp_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 _ppp_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 qppp_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 qppp_ _ 
2  q _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 qppp_ _ 
2  qq_ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 qppp_ _ 
2  qqq _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKp _ _
3 qpppq _ 
2  qqq _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKpq_ _
3 qpppq _ 
2  qqq _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4 qpKpq_ _
3 qpppq _ 
2  qqq _ k
1 _ _ _ _ 

  abcdefgh
Setup completed!
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ K _ _ 
4 qp pq_ _
3 qpppq _ 
2  qqq _ k
1 _ _ _ _ 

  abcdefgh
Final Score:
White: 0
Black: 0
